pkg/provider/kubernetes/crd/kubernetes_udp.go

```diff
@@ -78,6 +78,10 @@ func (p *Provider) loadIngressRouteUDPConfiguration(ctx context.Context, client
 func createLoadBalancerServerUDP(client Client, namespace string, service v1alpha1.ServiceUDP) (*dynamic.UDPService, error) {
```

Comment:
Would be nice to rename the `namespace` parameter into `parentNamespace` like you did in the createLoadBalancerServerTCP`. It's easier to understand where the namespace comes from.